README.md

```diff
@@ -14,99 +14,30 @@ Add your own `fetch` library.
 * for browsers use [whatwg-fetch](https://www.npmjs.com/package/whatwg-fetch) as polyfill for Internet Explorer
 * in AWS Lambda functions use [node-fetch](https://www.npmjs.com/package/node-fetch)
 
-> In browser enviroments
-> copy ` dist/bima-shark-sdk.js ` into your app.
-
+> In browser enviroments:
+> - run `npm run build`
+> - copy ` dist/bima-shark-sdk.js ` into your app.
```

Comment:
I would add the park about `npm link`, I think its more handy or 🤔 ?

Reply:
I found `npm link` to much of a hassle.
But for local testing I just installed the package by the absolute folder path.
```
cd bima-meine-bima-menu
npm install --save /Users/.../bima-shark-js-sdk/
```

Worked really well ...

Reply:
it work like charm as long as u remember to run the project 🤣